### DISCUSSION TITLE
Weekly playtime reward

### DISCUSSION THREAD

**Discussion by Buck-Irl** [gameserverapp/Platform#943] [Suggestions]
Afternoon Max.

Looking to add a weekly playtime reward , based on a player playing more then X hours.

I was looking at using the `Player played [X] hours in total` but there doesn't seem to be a way to reset that weekly, even with roles, without have to manually change the roll each week.

I am looking at the `Player voted [X] times [since]` and wondering could we get something like `Player played [X] hours in total [since]`

Or if I have completely missed an existing way to do this please let me know.

Buck

**Comment by mvaessen** [comment]
Thanks for the suggestion, this is now live!
